babble/src/main/java/io/mosaicnetworks/babble/configure/ArchivedGroupsFragment.java

```diff
@@ -49,7 +49,6 @@
 import java.util.Objects;
 
 import io.mosaicnetworks.babble.R;
-import io.mosaicnetworks.babble.node.BabbleConstants;
```

Comment:
Do we want to remove BabbleConstants from the project? 

Whilst it is only lightly used at the moment, removing it, when it is going to go back in post refactor seems retrograde, 

The fix to add BabbleConstants.getInstance(this) to XXXActivity.OnCreate will fix the disappearing static class instances.

Reply:
BabbleConstants has been removed from this class since it's only used to get the babbling port. In future we should not need to pass a port to babble in order to open an archive, in which case this class will not need a reference to BabbleConstants.